OpenRA.Mods.Cnc/Graphics/ClassicTilesetSpecificSpriteSequence.cs

```diff
@@ -59,7 +59,7 @@ public class ClassicTilesetSpecificSpriteSequence : ClassicSpriteSequence
 		public static bool AddExtension => true;
 
 		[Desc("Whether `mod.yaml`'s `TilesetExtensions` should be used with the sequence's file name.")]
-		public static bool UseTilesetExtension { get; private set; }
+		public bool UseTilesetExtension { get; private set; }
```

Comment:
Why can't this be `public static bool UseTilesetExtension => false;` like the rest? (And use `var useTilesetExtension = ...` like before instead of assigning to this field.)

Reply:
It could be. If we do this then we probably also want to remove the `static`s from `DefaultSpriteSequence` for consistency within each class.

Reply:
Sounds like we should go for it.